apps/Gruntfile.js

```diff
@@ -649,6 +649,8 @@ describe('entry tests', () => {
       './src/sites/studio/pages/programming_expressions/new.js',
     'programming_expressions/edit':
       './src/sites/studio/pages/programming_expressions/edit.js',
+    'reference_guides/show':
```

Comment:
nit: we learned recently that these teacher/student-facing pages should be in `codeStudioEntries` above